src/ansys/fluent/core/streaming_services/events_streaming.py

```diff
@@ -68,8 +70,8 @@ def _process_streaming(self, id, stream_begin_method, started_evt, *args, **kwar
 
     def register_callback(
         self,
-        event_name: Optional[str] = None,
-        call_back: Optional[Callable] = None,
+        event_name: str,
+        call_back: Callable,
```

Comment:
The original code here is wrong. The method name contains _callback_ but the argument `call_back` contains an underscore.

Reply:
Right. It should be `callback`. Thanks.

Reply:
Done. Thank you.